webapp/channels/src/components/post_view/post_attachment_container/post_attachment_container.tsx

```diff
@@ -48,6 +48,10 @@ const PostAttachmentContainer = (props: Props) => {
         const {tagName} = e.target;
         e.stopPropagation();
         const elements = ['A', 'IMG', 'BUTTON', 'I'];
+        if (e.target.closest('a')) {
```

Comment:
This code is already trying to do this with the if statement below, but it's just checking the type of the target element instead of checking if it's contained in any element of that type.

Could you change this to check each of the types in `elements` and replace the `!elements.includes(tagName)` below with an early return?

Reply:
Done !